### PR TITLE
Add support for settings idle_interval in connection URL

### PR DESCRIPTION
**Pull request by AndrewDryga**
Related change in Postgrex: https://github.com/elixir-ecto/postgrex/commit/031018f8b6953ab2ef1efb22da7ebaeb40572755

### CODE REVIEW

File: lib/ecto/repo/supervisor.ex [hunk 1]
```diff
@@ -58,8 +58,9 @@ defmodule Ecto.Repo.Supervisor do
     end
 
     if Code.ensure_compiled(adapter) != {:module, adapter} do
-      raise ArgumentError, "adapter #{inspect adapter} was not compiled, " <>
-                           "ensure it is correct and it is included as a project dependency"
+      raise ArgumentError,
+            "adapter #{inspect(adapter)} was not compiled, " <>
+              "ensure it is correct and it is included as a project dependency"
```
Comment (AndrewDryga) [inline lib/ecto/repo/supervisor.ex:63]: Should I roll back or keep formatter changes? They don't look so bad and I guess we want to apply formatter on codebase as we change it.

Comment (josevalim) [reply lib/ecto/repo/supervisor.ex:63]: No formatter changes, please. It makes it harder to review and messes up blame. :)

Comment (AndrewDryga) [reply lib/ecto/repo/supervisor.ex]: @josevalim true, I tell that to people all the time too but still did it for some reason 🤣. It's fixed!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**